configurations/kakuma_pipeline_config.json

```diff
@@ -26,6 +26,8 @@
     }
   ],
   "ListeningGroupCSVURLs":[
+    "gs://avf-project-datasets/2019/WUSC/LISTENING_GROUPS/repeat_listening_group.csv",
```

Comment:
Isn't this project called WUSC-KEEP-II rather than WUSC?

Reply:
yes have renamed to WUSC-KEEP-II thanks.